### PR TITLE
Adding a many to many field to the Project form

### LINKED ISSUE #198: Add a project-user ManyToMany field to the project form

### PR DESCRIPTION
**Pull request by livike**
For now it's only with a default SelectMultiple widget.
Closes #198

### CODE REVIEW

File: web/form.py [hunk 1]
```diff
@@ -69,7 +69,8 @@ class Meta:
 				'repository_link',
 				'website_production',
 				'website_test',
-				'project_owner',)
+				'project_owner',
+				'user_project',)
```
Comment (gandalfar) [inline web/form.py]: ) should be on it's own line, so:

  'user_project',
)

File: web/form.py [hunk 2]
```diff
@@ -69,7 +69,8 @@ class Meta:
...
```
Comment (ialja) [inline web/form.py]: I'd personally name this `project_members`, so it's more clear what the field refers to.

Comment (livike) [inline web/form.py]: Ok, will change and migrate

**Comment by ialja** [comment]
Merging, but it would also be nice to display all members on the project detail page.

**Pull request merged**